buildSrc/src/main/kotlin/buildsrc/tasks/ConvertCommonTestResourcesToKotlin.kt

```diff
@@ -0,0 +1,90 @@
+ *
+ * This is necessary for accessing test data in Kotlin Multiplatform tests, because there's
+ * currently no easy way to access and traverse filesystem content in Kotlin Multiplatform code.
+ */
+@CacheableTask
+abstract class ConvertCommonTestResourcesToKotlin @Inject constructor(
+    private val fs: FileSystemOperations,
+) : DefaultTask() {
+    /** The directory that will contain the generated Kotlin code. */
+    @get:OutputDirectory
+    abstract val destination: DirectoryProperty
+
+    // HACK: Ideally we'd retrieve the 'resources' path via Gradle API, but somehow Gradle sees only 'main' and 'test'
+    // source sets instead of the required 'commonTest'.
+    private val commonResourcesDir: File = project.rootProject.projectDir
+        .resolve("src").resolve("commonTest").resolve("resources")
+
+    @TaskAction
+    fun action() {
+        val destination = destination.asFile.get()
+        fs.delete { delete(destination) }
+        destination.mkdirs()
+
+        val resourcesMap = buildResourcesMap()
+        val code = generateKotlinCode(resourcesMap)
+
+        destination.resolve("CommonTestResources.kt").writeText(code)
+    }
+
+    private fun buildResourcesMap(): Map<String, Any> {
+        val resourcesMap = mutableMapOf<String, Any>()
+
+        commonResourcesDir.walk()
+            .filter { it.isFile }
+            .forEach { file ->
+                val relativePath = file.relativeTo(commonResourcesDir).toPath()
+                relativePath.toList().let { pathSegments: List<Path> ->
+                    val mapToAddFileTo = pathSegments
+                        .dropLast(1)
+                        .fold(resourcesMap) { acc, path ->
+                            @Suppress("UNCHECKED_CAST")
+                            acc.getOrPut(path.toString(), { mutableMapOf<String, Any>() }) as MutableMap<String, Any>
+                        }
+                    mapToAddFileTo[pathSegments.last().toString()] = file.readBytes()
+                }
+            }
+
+        return resourcesMap
+    }
+
+    private fun generateKotlinCode(resourcesMap: Map<String, Any>): String =
+        """
+            import okio.ByteString
+
+            object CommonTestResources {
+                val resourcesMap: Map<String, Any> = ${generateMapCode(resourcesMap)}
+            }
+        """.trimIndent()
+
+    private fun generateMapCode(map: Map<String, Any>): String {
+        return map.entries.joinToString(
+            separator = ",\n",
+            prefix = "mapOf(\n",
+            postfix = "\n)",
+        ) { (key, value) ->
+            @Suppress("UNCHECKED_CAST")
+            when (value) {
+                is ByteArray -> "\"$key\" to ByteString.of(${value.joinToString(separator = ", ") {
+                    "0x${"%02x".format(it)}.toByte()"
+                }})"
+                is Map<*, *> -> "\"$key\" to ${generateMapCode(value as Map<String, Any>)}"
```

Comment:
note: looks like the formatting will be broken in case of recursion call. Maybe, we should use something like `kotlin-poet` to generate correctly formatted code (I might take a look at that after this change is merged)

Reply:
You're right, the generated file looks like this:

```kotlin
            import okio.ByteString

            object CommonTestResources {
                val resourcesMap: Map<String, Any> = mapOf(
"issues" to mapOf(
"issue1086-2-input.yaml" to ByteString.of(0x26.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x32.toByte(), 0x0a.toByte(), 0x62.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x6b.toByte(), 0x41.toByte(), 0x63.toByte(), 0x63.toByte(), 0x6f.toByte(), 0x75.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x4f.toByte(), 0x77.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x6b.toByte(), 0x41.toByte(), 0x63.toByte(), 0x63.toByte(), 0x6f.toByte(), 0x75.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x4f.toByte(), 0x77.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x50.toByte(), 0x6c.toByte(), 0x61.toByte(), 0x63.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x4c.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x69.toByte(), 0x6e.toByte(), 0x67.toByte(), 0x72.toByte(), 0x61.toByte(), 0x64.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x64.toByte(), 0x61.toByte(), 0x79.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x31.toByte(), 0x39.toByte(), 0x37.toByte(), 0x30.toByte(), 0x2d.toByte(), 0x30.toByte(), 0x31.toByte(), 0x2d.toByte(), 0x31.toByte(), 0x32.toByte(), 0x54.toByte(), 0x31.toByte(), 0x33.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x36.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x30.toByte(), 0x5a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x63.toByte(), 0x68.toByte(), 0x69.toByte(), 0x6c.toByte(), 0x64.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x33.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x2d.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x32.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x2d.toByte(), 0x20.toByte(), 0x62.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x6b.toByte(), 0x41.toByte(), 0x63.toByte(), 0x63.toByte(), 0x6f.toByte(), 0x75.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x4f.toByte(), 0x77.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x50.toByte(), 0x6c.toByte(), 0x61.toByte(), 0x63.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x4e.toByte(), 0x65.toByte(), 0x77.toByte(), 0x20.toByte(), 0x59.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x6b.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x64.toByte(), 0x61.toByte(), 0x79.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x31.toByte(), 0x39.toByte(), 0x38.toByte(), 0x33.toByte(), 0x2d.toByte(), 0x30.toByte(), 0x34.toByte(), 0x2d.toByte(), 0x32.toByte(), 0x34.toByte(), 0x54.toByte(), 0x30.toByte(), 0x32.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x30.toByte(), 0x3a.toByte(), 0x30.toByte(), 0x30.toByte(), 0x5a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x63.toByte(), 0x68.toByte(), 0x69.toByte(), 0x6c.toByte(), 0x64.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x5b.toByte(), 0x5d.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x66.toByte(), 0x61.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6d.toByte(), 0x6f.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x34.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x6b.toByte(), 0x41.toByte(), 0x63.toByte(), 0x63.toByte(), 0x6f.toByte(), 0x75.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x4f.toByte(), 0x77.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x50.toByte(), 0x6c.toByte(), 0x61.toByte(), 0x63.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x53.toByte(), 0x61.toByte(), 0x69.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x2d.toByte(), 0x50.toByte(), 0x65.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x73.toByte(), 0x62.toByte(), 0x75.toByte(), 0x72.toByte(), 0x67.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x64.toByte(), 0x61.toByte(), 0x79.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x31.toByte(), 0x39.toByte(), 0x37.toByte(), 0x33.toByte(), 0x2d.toByte(), 0x30.toByte(), 0x33.toByte(), 0x2d.toByte(), 0x30.toByte(), 0x33.toByte(), 0x54.toByte(), 0x30.toByte(), 0x39.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x36.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x30.toByte(), 0x5a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x63.toByte(), 0x68.toByte(), 0x69.toByte(), 0x6c.toByte(), 0x64.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x33.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x66.toByte(), 0x61.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6d.toByte(), 0x6f.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x4d.toByte(), 0x6f.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x70.toByte(), 0x61.toByte(), 0x72.toByte(), 0x74.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x44.toByte(), 0x61.toByte(), 0x75.toByte(), 0x67.toByte(), 0x68.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x70.toByte(), 0x61.toByte(), 0x72.toByte(), 0x74.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x66.toByte(), 0x61.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6d.toByte(), 0x6f.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x46.toByte(), 0x61.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x70.toByte(), 0x61.toByte(), 0x72.toByte(), 0x74.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x34.toByte(), 0x0a.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x50.toByte(), 0x6c.toByte(), 0x61.toByte(), 0x63.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x4d.toByte(), 0x75.toByte(), 0x6e.toByte(), 0x69.toByte(), 0x63.toByte(), 0x68.toByte(), 0x0a.toByte(), 0x62.toByte(), 0x69.toByte(), 0x72.toByte(), 0x74.toByte(), 0x68.toByte(), 0x64.toByte(), 0x61.toByte(), 0x79.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x31.toByte(), 0x39.toByte(), 0x37.toByte(), 0x39.toByte(), 0x2d.toByte(), 0x31.toByte(), 0x30.toByte(), 0x2d.toByte(), 0x32.toByte(), 0x38.toByte(), 0x54.toByte(), 0x32.toByte(), 0x33.toByte(), 0x3a.toByte(), 0x30.toByte(), 0x36.toByte(), 0x3a.toByte(), 0x34.toByte(), 0x30.toByte(), 0x5a.toByte(), 0x0a.toByte(), 0x63.toByte(), 0x68.toByte(), 0x69.toByte(), 0x6c.toByte(), 0x64.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x5b.toByte(), 0x5d.toByte(), 0x0a.toByte(), 0x66.toByte(), 0x61.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x6d.toByte(), 0x6f.toByte(), 0x74.toByte(), 0x68.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x69.toByte(), 0x64.toByte(), 0x30.toByte(), 0x30.toByte(), 0x34.toByte(), 0x0a.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x53.toByte(), 0x6f.toByte(), 0x6e.toByte(), 0x0a.toByte(), 0x70.toByte(), 0x61.toByte(), 0x72.toByte(), 0x74.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x75.toByte(), 0x6c.toByte(), 0x6c.toByte(), 0x0a.toByte()),
"issue1086-1-expected.yaml" to ByteString.of(0x64.toByte(), 0x65.toByte(), 0x66.toByte(), 0x69.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x73.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x74.toByte(), 0x79.toByte(), 0x70.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x74.toByte(), 0x33.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x74.toByte(), 0x72.toByte(), 0x61.toByte(), 0x67.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x35.toByte(), 0x30.toByte(), 0x30.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x32.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x74.toByte(), 0x79.toByte(), 0x70.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x74.toByte(), 0x32.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x74.toByte(), 0x72.toByte(), 0x61.toByte(), 0x67.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x32.toByte(), 0x35.toByte(), 0x30.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x46.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x50.toByte(), 0x75.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x69.toByte(), 0x63.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x32.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x46.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x49.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x63.toByte(), 0x75.toByte(), 0x72.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x61.toByte(), 0x73.toByte(), 0x73.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x65.toByte(), 0x64.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x74.toByte(), 0x79.toByte(), 0x70.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x74.toByte(), 0x33.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x74.toByte(), 0x72.toByte(), 0x61.toByte(), 0x67.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x35.toByte(), 0x30.toByte(), 0x30.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x46.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x50.toByte(), 0x75.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x69.toByte(), 0x63.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x31.toByte(), 0x0a.toByte()),
"issue1086-1-input.yaml" to ByteString.of(0x64.toByte(), 0x65.toByte(), 0x66.toByte(), 0x69.toByte(), 0x6e.toByte(), 0x65.toByte(), 0x73.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x5f.toByte(), 0x48.toByte(), 0x69.toByte(), 0x67.toByte(), 0x68.toByte(), 0x50.toByte(), 0x65.toByte(), 0x72.toByte(), 0x66.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x6d.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x63.toByte(), 0x65.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x74.toByte(), 0x79.toByte(), 0x70.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x74.toByte(), 0x33.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x74.toByte(), 0x72.toByte(), 0x61.toByte(), 0x67.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x35.toByte(), 0x30.toByte(), 0x30.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x32.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x5f.toByte(), 0x4c.toByte(), 0x6f.toByte(), 0x77.toByte(), 0x50.toByte(), 0x65.toByte(), 0x72.toByte(), 0x66.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x6d.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x63.toByte(), 0x65.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x74.toByte(), 0x79.toByte(), 0x70.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x74.toByte(), 0x32.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x74.toByte(), 0x72.toByte(), 0x61.toByte(), 0x67.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x32.toByte(), 0x35.toByte(), 0x30.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x5f.toByte(), 0x50.toByte(), 0x75.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x69.toByte(), 0x63.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x46.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x50.toByte(), 0x75.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x69.toByte(), 0x63.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x50.toByte(), 0x61.toByte(), 0x74.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x32.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x26.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x5f.toByte(), 0x49.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6d.toByte(), 0x65.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x46.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x49.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x65.toByte(), 0x72.toByte(), 0x6e.toByte(), 0x61.toByte(), 0x6c.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x76.toByte(), 0x70.toByte(), 0x63.toByte(), 0x31.toByte(), 0x0a.toByte(), 0x63.toByte(), 0x75.toByte(), 0x72.toByte(), 0x72.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x74.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x61.toByte(), 0x73.toByte(), 0x73.toByte(), 0x65.toByte(), 0x6e.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x65.toByte(), 0x64.toByte(), 0x31.toByte(), 0x3a.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x73.toByte(), 0x65.toByte(), 0x72.toByte(), 0x76.toByte(), 0x65.toByte(), 0x72.toByte(), 0x5f.toByte(), 0x48.toByte(), 0x69.toByte(), 0x67.toByte(), 0x68.toByte(), 0x50.toByte(), 0x65.toByte(), 0x72.toByte(), 0x66.toByte(), 0x6f.toByte(), 0x72.toByte(), 0x6d.toByte(), 0x61.toByte(), 0x6e.toByte(), 0x63.toByte(), 0x65.toByte(), 0x0a.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x20.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x3a.toByte(), 0x20.toByte(), 0x2a.toByte(), 0x6c.toByte(), 0x62.toByte(), 0x5f.toByte(), 0x50.toByte(), 0x75.toByte(), 0x62.toByte(), 0x6c.toByte(), 0x69.toByte(), 0x63.toByte(), 0x0a.toByte())
)
)
            }
```

but it's a valid Kotlin and it already provides some value. I thought about Kotlin Poet, but given that the code is fairly simple, I'm wondering if we want to add a dependency on the library. For now I'd even keep the weird formatting and keep the code for generating Kotlin code simpler.